routes/nodeMailer.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+
+const emailRouter = express();
+const transporter = require('../transporter');
+
+emailRouter.use(express.json());
+
+emailRouter.post('/', async (req, res) => {
+  try {
+    const { email, htmlMessage } = req.body;
+
+    const mail = {
+      from: `${process.env.REACT_APP_EMAIL_FIRST_NAME} ${process.env.REACT_APP_EMAIL_LAST_NAME} ${process.env.REACT_APP_EMAIL_USERNAME}`,
```

Comment:
Could you add the values for these environment variables to the [backend .env](https://docs.google.com/document/d/1iWBERH8V9rV6Ha6OnqZUlIDslyiU46JXFM2jgnf6UQs/edit) so that I can test it?